src/Compilers/Core/Portable/Symbols/Nullability.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis
+{
+    public enum Nullability
+    {
+        NonNull,
+        CanBeNull,
+        Unknown
```

Comment:
The `enum` values should probably be explicit, and it feels like `Unknown` should be `0`.